kahuna/public/js/downloader/downloader.html

```diff
@@ -0,0 +1,19 @@
+<button class="download"
+        type="button"
+        title="Download images"
+        ng:disabled="ctrl.downloading"
+        ng:if="ctrl.images.size > 1"
+        ng:click="ctrl.download()">
+    <gr-icon-label gr-icon="file_download">
+        Download<span ng:show="ctrl.downloading">ing</span>
+    </gr-icon-label>
+</button>
+
+<a class="download"
+   ng:if="ctrl.images.size == 1"
+   ng:disabled="ctrl.downloading"
```

Comment:
Its a possibility, there are some pretty big images and some pretty slow connections...